extension/src/plots/vega/util.ts

```diff
@@ -145,15 +148,23 @@ export const getSpecEncodingUpdate = ({
   if (strokeDash) {
     encoding.strokeDash = {
       ...strokeDash,
-      legend: { disable: true }
+      legend: {
+        disable: true,
+        symbolFillColor: 'transparent',
+        symbolStrokeColor: 'grey'
```

Comment:
Should these be set where it's enabled only? (Line 311)

Reply:
I am going to have to change the approach slightly now that we don't know exactly where the encoding update has been made.

Reply:
I will call out the reasoning for leaving in the follow-up.